ektelo/plans/standalone.py

```diff
@@ -717,42 +706,44 @@ def __init__(self, ratio, rounds, data_scale, domain_shape, total_noise_scale):
 
     def Run(self, W, x, eps, seed):
         prng = np.random.RandomState(seed)
+
         domain_size = np.prod(self.domain_shape)
         # Start with a unifrom estimation of x
         x_hat = np.array([self.data_scale / float(domain_size)] * domain_size)
             
-        W = get_matrix(W)
-
-        W_partial = sparse.csr_matrix(W.shape)
         nnls = inference.NonNegativeLeastSquares()
 
-        M_history = np.empty((0, domain_size))
+        measuredQueries = []
+        M_history = []
         y_history = []
+        noise_scales = []
+
+        if self.total_noise_scale != 0:
```

Comment:
What's happening here? Is this something that @horseno added for the reproducibility effort?

Reply:
Yes, I added this while trying to reproduce the mwem variant, but it's not necessarily for the reproducibility effort. It's adding support for total_noise_scale which is an optional plan parameter.